### ISSUE TITLE
50java.run: LC_CTYPE: cannot change locale (en): No such file or directory

### ISSUE THREAD

**Issue by jirislaby** [openSUSE/ca-certificates#6]
During update of java, I see:
```
Executing %posttrans script 'java-11-openjdk-headless-11.0.4.0-1.2.x86_64.rpm' ............................................<60%>=================================[/]
Output of java-11-openjdk-headless-11.0.4.0-1.2.x86_64.rpm %posttrans script:
    /usr/lib/ca-certificates/update.d/50java.run: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en): No such file or directory
    /usr/lib/ca-certificates/update.d/50java.run: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en)
```

**Comment by jirislaby** [comment]
Don't you want to use `POSIX` instead?

**Comment by lnussel** [comment]
missed this one :/ TBH I have no idea why I used en_US. I guess there was a reason

**Comment by lnussel** [comment]
was actually fixed long time ago in 9c933b98b8a0a81ea2b110381b14e39be6d91440
